.ci/bump-stack-version.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+#
+# Given the stack version this script will bump the version.
```

Comment:
It seems that the same script will land in multiple places. Is it possible to keep it in a single public one instead of maintaining multiple copies?

Reply:
I see more cons:
1) Each project might differ.
2) The bump script will not be easy to access and all the issues related to how to test, how to version.
3) Each project branch might potentially differ in how to bump those versions, even if not now it could happen

Maybe the script should be moved to the build system instead, and delegate that particular functionality to the project itself. But that's something we can iterate on it in the future if needed.

We can revisit this in the future to refactor if needed, what do you think?